lib/bundler/audit/cli.rb

```diff
@@ -60,10 +61,10 @@ def check
       end
 
       desc 'update', 'Updates the ruby-advisory-db'
-      def update
+      def update(quiet = false)
```

Comment:
In my own projects, I would normally make this a keyword arg but the README says you are trying to support Ruby 1.9.3 which doesn't have them. Thus this is what I went with.